src/modules/cli/hooks/CommingSoon/useCommingSoon.ts

```diff
@@ -0,0 +1,61 @@
+import { useDisclosure, UseDisclosureReturn } from '@chakra-ui/react';
+import { CLIFeaturesLabels } from '../useCLI';
+
+export interface FeatureConfig {
+  dialogDescription: string;
+  gtmEventName: string;
+  gtmButtonId: string;
+  gtmDescription: string;
+  notifyHandler: () => void;
+}
+
+export interface UseCommingSoonProps {
+  commingSoonDialog: UseDisclosureReturn;
+  features: {
+    [key in CLIFeaturesLabels]: FeatureConfig;
+  };
+}
+
+const useCommingSoon = () => {
+  const commingSoonDialog = useDisclosure();
+
```

Comment:
create a fn with:

`
handleAction(field: CLIFeaturesLabels) => {
switch 
 case CLIFeaturesLabels.ADD_OTHER_TOKENS:
   createGTMCustomEvent({
   ...params
   })
   
   ...other cases
   and call notifyHandler()
}
`

Reply:
Done